pendulum_control/scripts/pendulum_launch.bash

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+pendulum_logger__rmw_opensplice_cpp &
+logger_pid=$!
+
+# run pendulum demo indefinitely
+pendulum_demo__rmw_opensplice_cpp -i 0 &
+#pendulum_demo__rmw_opensplice_cpp -i 0
```

Comment:
Are you using these commented command frequently for testing? If not I would suggest to remove them (same for last line in this file).